front/src/common/Map/Search/MapSearchStation.jsx

```diff
@@ -89,16 +91,16 @@ export default function MapSearchStation(props) {
     }
   };
 
-  // Sort on name, and on yardname
-  const orderResults = (results) =>
-    results.sort((a, b) => a.name.localeCompare(b.name) || a.ch.localeCompare(b.ch));
-
   const clearSearchResult = () => {
     setSearch('');
     setSearchResults(undefined);
     dispatch(updateMapSearchMarker(undefined));
   };
 
+  // Sort on name, and on yardname
+  const orderResults = (results) =>
+    results.sort((a, b) => a.name.localeCompare(b.name) || a.ch.localeCompare(b.ch));
```

Comment:
You can order the results when you receive it from the API (in updateSearch)

Reply:
It is refacto in another PR